src/App.js

```diff
@@ -8,8 +9,16 @@ const article = {
   date: 'August 30, 2019',
 };
 
-const App = () => (
-  <h1>React Article</h1>
-);
+function App() {
+  return (
+    <>
+      <Article
+        title={article.title}
+        text={article.text}
+        date={article.date}
+      />
```

Comment:
```suggestion
      <Article {...article} />
```
You can just use destructurisation

Reply:
давай використаємо спред оператор